docs/tutorials/setup-typescript.md

```diff
@@ -0,0 +1,41 @@
+Atomist Rugs are most often developed using [TypeScript][ts].  To
+successfully complete the rest of the tutorials, you will need to have
+several TypeScript-related tools installed locally on your system.
+
+[ts]: https://www.typescriptlang.org/ (TypeScript)
+
+## NPM
+
+You will need the [Node.js package manager][npm], `npm`, installed on
+your system to install the Rug TypeScript dependencies.  The easiest
+way to install NPM is to install [Node.js][node], whose installation
+includes NPM.  There are links to install Node.js right on its home
```

Comment:
Consider removing sentence: `There are links to install Node.js right on its home page`. The homepage is linked above, so this seems superfluous to me. Also, the use of 'right' reads a little strangely to me.